CHANGELOG.md

```diff
@@ -59,6 +59,10 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 ## Changes for v1.2.x
 
+### Bug Fixes
+
```

Comment:
Leave out this extra header because we don't distinguish types of updates in patch releases.

Reply:
thanks! reflected in the latest commit